Source/Commands/LambdaBodyCommand.ts

```diff
@@ -0,0 +1,112 @@
+/// <reference path="../Languages/Language.ts" />
+/// <reference path="Command.ts" />
+/// <reference path="LineResults.ts" />
+/// <reference path="Parameters/Parameter.ts" />
+/// <reference path="Parameters/SingleParameter.ts" />
+/// <reference path="Parameters/RepeatingParameters.ts" />
+
+namespace GLS.Commands {
+    "use strict";
+
+    /**
+     * A command for a lambda function body.
+     */
+    export class LambdaBodyCommand extends Command {
+        /**
+         * Information on parameters this command takes in.
+         */
+        private static parameters: Parameters.Parameter[] = [
+            /*
+            new Parameters.SingleParameter("returnType", "The return type of the function.", true),
+            */
+            new Parameters.SingleParameter(
+                "returnType",
+                "Return type of the lambda function",
+                true),
+            new Parameters.RepeatingParameters(
+                "Lambda function parameters.",
+                [
+                    new Parameters.SingleParameter(
+                        "parameterName",
+                        "A named parameter for the lambda function.",
+                        true),
+                    new Parameters.SingleParameter(
+                        "parameterType",
+                        "The type of the parameter.",
+                        true)
+                ]),
+            new Parameters.SingleParameter(
+                "functionBody",
+                "The actual body of the lambda function",
+                true)
+        ];
+
+        /**
+         * @returns Information on parameters this command takes in.
+         */
+        public getParameters(): Parameters.Parameter[] {
+            return LambdaBodyCommand.parameters;
+        }
+
+        /**
+         * Renders the lambda for a language with the given parameters.
+         *     GLS syntax (order of parameters) lambda returnType [parameterName, parameterType, ...] command
+         *     Render format: lambdaLeft parameterType parameterName, ... lambdaMiddle commandString lambdaRight
+         * @param parameters   The command's name, followed by any parameters.
+         * @returns Line(s) of code in the language.
+         * @remarks Usage: ([parameterName, parameterType, ...]).
+         */
+        public render(parameters: string[]): LineResults {
+            let returnType: string = this.context.convertCommon("type", parameters[2]),
+                lambdaBody: string = "",
+                output: CommandResult[];
+
+            lambdaBody += this.language.properties.lambdas.functionLeft;
+
+            if (this.language.properties.lambdas.returnTypeRequired) {
+                // No language requires this yet, so format is unknown.
+                throw Error("returnTypeRequired=true not implemented");
+            }
+
+            // If we have at least 1 set of parameters, need to handle them.
```

Comment:
Similar to getting rid of the big chunks of code: there's no need to have these inline comments. The code is self-explanatory.